x/wasm/internal/keeper/keeper.go

```diff
@@ -42,6 +42,10 @@ const InstanceCost uint64 = 40_000
 // CompileCost is how much SDK gas we charge *per byte* for compiling WASM code.
 const CompileCost uint64 = 2
 
+// contractMemoryLimit is the memory limit of each contract execution (in MiB)
+// constant value so all nodes run with the same limit.
+const contractMemoryLimit = 32
```

Comment:
Let's make this a param in a later PR.
If not too much work, we can add that as a param (with default 32 MB) in a separate PR, but include that in the 0.14 release.

I am happy to merge this first and test it manually before that addition. (It can come later as well)

Reply:
After reflection, that params issue is part of #202 
Let's just do that with everything else